R/AllGenerics.R

```diff
@@ -569,19 +569,30 @@ setGeneric("family<-", function(object, value) standardGeneric("family<-"))
 #' [XCMSnExp-class] object. The function returns for each feature the
 #' extracted ion chromatograms (along with all associated chromatographic
 #' peaks) in each sample. The chromatogram is extracted from the m/z - rt
-#' region including all chromatographic peaks of that features (i.e. based on
-#' the ranges of `"mzmin"`, `"mzmax"`, `"rtmin"`, `"rtmax"` of all
-#' chromatographic peaks of the feature).
+#' region that includes **all** chromatographic peaks of a feature. By default,
```

Comment:
This is the updated documentation of the `featureChromatograms` function, explaining that the EICs are *globally* defined for a feature, i.e. the EIC is extracted for the same m/z - rt area from all samples. The EIC in a specific sample does thus not necessarily represent *exactly* the chromatographic peak identified in that sample